iOS_SDK/OneSignalSDK/Source/OSPermission.m

```diff
@@ -46,22 +46,22 @@ - (instancetype)initAsTo {
 
 - (instancetype)initAsFrom {
     let standardUserDefaults = OneSignalUserDefaults.initStandard;
-    _hasPrompted = [standardUserDefaults getSavedBoolForKey:PERMISSION_HAS_PROMPTED defaultValue:false];
-    _answeredPrompt = [standardUserDefaults getSavedBoolForKey:PERMISSION_ANSWERED_PROMPT defaultValue:false];
-    _accepted  = [standardUserDefaults getSavedBoolForKey:PERMISSION_ACCEPTED defaultValue:false];
-    _provisional = [standardUserDefaults getSavedBoolForKey:PERMISSION_PROVISIONAL_STATUS defaultValue:false];
-    _providesAppNotificationSettings = [standardUserDefaults getSavedBoolForKey:PERMISSION_PROVIDES_NOTIFICATION_SETTINGS defaultValue:false];
+    _hasPrompted = [standardUserDefaults getSavedBoolForKey:OSUD_WAS_PROMPTED_FOR_NOTIFICATIONS_LAST defaultValue:false];
+    _answeredPrompt = [standardUserDefaults getSavedBoolForKey:OSUD_PERMISSION_ANSWERED_PROMPT defaultValue:false];
```

Comment:
Add `_LAST` to these 4 `initAsFrom` and `persistAsFrom` keys

Reply:
Got it!